frontend/test/resolution/testCPtr.cpp

```diff
@@ -245,6 +245,52 @@ static void test17() {
   });
 }
 
+static void test18() {
+  testCPtrArg("c_ptrConst(int)", "c_ptr(int)", [](const TypedFnSignature* fn, const CPtrType* t, ErrorGuard& eg) {
+    assert(t);
```

Comment:
should this test also assert that the `TypedFnSignature` is found?

Reply:
I think `formalTypePtr` couldn't be non-null if `fn` is null, but I'll add the check anyway.